portal-impl/src/com/liferay/portal/spring/aop/ServiceBeanAutoProxyCreator.java

```diff
@@ -62,8 +63,7 @@ protected Object wrapIfNecessary(
 
 		ProxyFactory proxyFactory = new ProxyFactory();
 
-		evaluateProxyInterfaces(beanClass, proxyFactory);
-
+		proxyFactory.setInterfaces(ReflectionUtil.getInterfaces(bean));
```

Comment:
ReflectionUtil.getInterfaces() doesn't have the same result as spring's lookup.  Spring also includes extended interfaces.